src/components/Cards/Recipe-Card/Recipe-Card.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export default function RecipeCard(props) {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg">
+      <img
+        className="w-full"
+        src={require('./img/breakfast3.png')}
+        alt="breakfast"
+      />
+      <div></div>
+      <div className="px-6 py-4 flex flex-col justify-around items-center ">
+        <h1 className="text-black-700 text-base font-bold text-xl mb-2 text-center">
+          Kiwi, Greek yogurt <br />& blueberries
+        </h1>
+        <button
+          className="bg-primary text-white active:bg-pink-600 font-poppins  text-sm px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mx-4"
+          type="button"
+          style={{ transition: 'all .15s ease' }}
+        >
+          See Recipe
+        </button>
+
+        <div className="my-4 flex flex-wrap justify-around items-center">
+          <div className=" flex flex-col justify-around items-center w-1/3">
+            <img
+              src={require('./img/calories-icon.png')}
```

Comment:
I don't think you need to use `require` here

Reply:
Ohhhh... it didn't work Osama, me & Evan spent alot of time trying to show the images but they didn't till we had this solution.. we had two actually either to require or import the image to show it... as I got from stackflow where we found this one. it says that the used webpack sometimes doesn't read from local source

Reply:
so with react, yes you have to import the image and add it in the source

Reply:
but I'm gonna use it's source from the recieved prop!!!! how to import & read it as a prop together is that possible???